src/pal/src/misc/errorstrings.cpp

```diff
@@ -0,0 +1,224 @@
+//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information. 
+//
+
+/*++
+
+
+
+Module Name:
+
+    errorstrings.cpp
+
+Abstract:
+
+    Conversion of PAL error code to string
+
+Revision History:
```

Comment:
I meant the "Revision History" section in particular. If it's just for consistency, that's fine.